src/openrct2/editor.c

```diff
@@ -528,3 +529,174 @@ bool editor_check_park()
 
 	return true;
 }
+
+void game_command_edit_scenario_options(sint32* eax, sint32* ebx, sint32* ecx, sint32* edx, sint32* esi, sint32* edi, sint32* ebp)
+{
+	if (*ebx & GAME_COMMAND_FLAG_APPLY) {
```

Comment:
@duncanspumpkin Honestly man... I want to organize it better, but i'm pretty clueless on where i would put many of these functions or what i'd name them., and i feel adding a new "editor/scenario_options" file would be pointless. Would you be able to help me out here?
1) for names, lets use the set no money as a reference example, should i do "park_flags_set_no_money" or "edit_scenario_options_set_no_money"? what would be a good name?
2) would i put set_no_money in `management/finances.h`, `world/park.h`, `scenario/scenario.h`, or `editor/scenario_options.h`?
3) the ones about forbidden actions / difficulties, "set_forbid_marketing_campaigns", "set_forbid_tree_removal", "set_forbid_landscape_changes", "set_forbid_high_construction", "set_park_rating_higher_difficult_level", "set_guest_generation_higher_difficult_level", where would i store these? i feel it would make sense to have them all in a similar file as they're geared to challenges.. heck, maybe even making an index enum for the certain forbids.